docs/available_checks.md

```diff
@@ -57,14 +57,14 @@ This document is a list of tables with a description and link to documentation f
 | [MixedCaseNameCheck](checks/mixedCaseNameCheck.md) | The purpose of this check is to identify names that contain invalid mixed cases so that they can be edited to be the standard format. |
 | [RoadNameSpellingConsistencyCheck](checks/RoadNameSpellingConsistencyCheck.md) | The purpose of this check is to identify road segments that have a name Tag with a different spelling from that of other segments of the same road. This check is primarily meant to catch small errors in spelling, such as a missing letter, letter accent mixups, or capitalization errors. |
 | [StreetNameIntegersOnlyCheck](checks/streetNameIntegersOnlyCheck.md) | The purpose of this check is to identify streets whose names contain integers only. |
-| UnusualLayerTagsCheck | The purpose of this check is to identify layer tag values when accompanied by invalid tunnel and bridge tags. |
+| [UnusualLayerTagsCheck](checks/unusualLayerTagsCheck.md) | The purpose of this check is to identify layer tag values when accompanied by invalid tunnel and bridge tags. |
 
 ## Ways
 | Check Name | Check Description |
 | :--------- | :---------------- |
 | [BuildingRoadIntersectionCheck](checks/buildingRoadIntersectionCheck.md) | The purpose of this check is to identify buildings that intersect/overlap roads. |
 | [~~DuplicateWaysCheck~~ (Deprecated)](checks/duplicateWaysCheck.md) | The purpose of this check is to identify Ways that have either had their entire length or a segment of their length duplicated or drawn multiple times. **This check has been deprecated and is no longer active.** |
 | [GeneralizedCoastlineCheck](checks/generalizedCoastlineCheck.md) | The purpose of this check is to identify coastlines whose nodes are too far apart. |
```

Comment:
For GeneralizedCoastlineCheck, its purpose has recently been ahem.. more generalized. We could say "The purpose is to identify poorly constructed coastlines that don't capture the real geometrical details"

Reply:
Thanks Sean.  I agree that check documentation should always be updated to be accurate whenever any changes to check functionality are being made.  When you say "real geometrical details," are you referring to the ground-truth coastline?

Reply:
That's right, the geometry of the real world/ground truth coastline

Reply:
Just updated the description for GeneralizedCoastlineCheck to also include sharp angles and source=PGS Tag values.  :)